backend/external/uploadImage.js

```diff
@@ -0,0 +1,26 @@
+const imgbb = require('./imgbb')
+const imgur = require('./imgur')
+
+const uploadImage = async (base64) => {
+
+  if (!base64) {
+    throw new Error('base64 - required arg');
+  }
+  
+  const IMGUR = 'imgur'
+  const IMGBB = 'imgbb'
+
+  const IMAGE_HOSTING_PROVIDER = IMGBB
```

Comment:
Not sure if those const vars are required.
What about just: 
```suggestion
  const IMAGE_HOSTING_PROVIDER = 'imgbb'
```